examples/demo-react-native-jest/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "detox": {
     "test-runner": "jest",
+    "runner-config": "e2e/config.json",
```

Comment:
The default value is `e2e/config.json` already. I suggested above to extract all this runner-config "explicitification" to another PR, if you like.

Reply:
I added a simplified note in the docs, and will have this removed.